python/ucxx/ucxx/_lib_async/continuous_ucx_progress.py

```diff
@@ -70,3 +77,133 @@ async def _progress_task(self):
             worker.progress()
             # Give other co-routines a chance to run.
             await asyncio.sleep(0)
+
+
+class BlockingMode(ProgressTask):
+    def __init__(
+        self,
+        worker: UCXWorker,
+        event_loop: asyncio.AbstractEventLoop,
+        progress_timeout: float = 1.0,
+    ):
+        """Progress the UCX worker in blocking mode.
+
+        The blocking progress mode ensure the worker is progress whenever the UCX
+        worker reports an event on its epoll file descriptor. In certain
+        circumstances the epoll file descriptor may not
```

Comment:
*nit*: Grammar suggestion:

> Blocking progress mode ensures that the worker is progressed whenever the UCX worker reports and event on its epoll file descriptor.

*nit8: The second sentence appears to be incomplete.